### ISSUE TITLE
How to save the model?

### ISSUE THREAD

**Issue by mirfan899** [oulenz/fuzzy-rough-learn#7]
I tried to save the model using pickle but getting the following errors on the FROVOCO model.

```python

frovoco_dict = {
    "model": frovoco,
    "tfidf": data_vectorizer
}

pickle.dump(frovoco_dict, open("models/frovoco.pickle", "wb"))
```

Error
```shell
Traceback (most recent call last):
  File "/home/irfan/PycharmProjects/FuzzyClasification/frovoco_tfidf.py", line 33, in <module>
    pickle.dump(fronec_dict, open("models/frovoco.pickle", "wb"))
AttributeError: Can't pickle local object 'exponential.__init__.<locals>.<lambda>'
```

**Comment by mirfan899** [comment]
any update?

**Comment by oulenz** [comment]
Sorry, yes. The problem seems to be that pickle can't automatically handle lambda functions, see discussions like these:

https://stackoverflow.com/questions/25348532/can-python-pickle-lambda-functions

(An easy way to get it to work is apparently by importing `dill`.)

I'm also working on new version of fuzzy-rough-learn (0.2) that I hope to release in the coming weeks, that no longer uses lambda functions.

**Comment by oulenz** [comment]
I've just released version 0.2.0.
